### PR TITLE
Celo blockchain machine images

### PR DESCRIPTION
**Pull request by AlexBHarley**
### Description

Automated machine image creation for the major clouds (AWS, GCP and Azure) via Packer.

Leverages some existing scripts in the `terraform-modules-public` folder.

### Other changes

N/A

### Tested

A bunch via creating VM's on various clouds.

### Related issues

N/A

### Backwards compatibility

N/A

### CODE REVIEW

**Review by tkporter — APPROVED** [review]
I've never used packer but at the surface all looks good

Should there be a section for baklava images in the makefile as well?

**Comment by AlexBHarley** [comment]
@tkporter good q. I can't imagine why people want to run full nodes on Baklava right now but it's easy enough to add.

Will get to it this at some point in the next week.

File: packages/docs/getting-started/hosted-nodes.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Hosted nodes
+
+This page explains how to get a preconfigured Celo blockchain node running on one of the major cloud providers. cLabs currently provides machine images for launching full and lightest nodes on Alfajores and Mainnet, these prebuilt images are updated with every release of the Celo blockchain client and available on Amazon Web Services and Google Cloud Platform.
+
+Before proceeding with a hosted Celo blockchain node, you'll need to have an account with your cloud provider of choice and basic knowledge of networking.
+
+{% hint style="info" %}
+If you would like to keep up-to-date with all the news happening in the Celo community, including validation, node operation and governance, please sign up to our [Celo Signal mailing list here](https://celo.activehosted.com/f/15).
```
Comment (barbaraliau) [inline packages/docs/getting-started/hosted-nodes.md]: nit: "sign up ~to~ for our [Celo Signal mailing list ~here~"

"here" is unnecessary.

File: packages/docs/getting-started/hosted-nodes.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+You can add the [Celo Signal public calendar](https://calendar.google.com/calendar/u/0/embed?src=c_9su6ich1uhmetr4ob3sij6kaqs@group.calendar.google.com) as well which has relevant dates.
+{% endhint %}
+
+## Amazon Web Services
+
+To get started with a Celo blockchain node on AWS, ensure you're in the North Virginia region (us-east-1). You'll need to navigate to the AMIs tab of the EC2 page. From there you can change your search to only include `Public images` and enter `celo-`.
+
+The cLabs AWS owner ID is `243983831780`, if you're new to Celo or cloud providers, ensure the `Owner` of an AMI you launch matches `243983831780`.
```
Comment (barbaraliau) [inline packages/docs/getting-started/hosted-nodes.md:18]: Should we make this into a hint so it's prominent?

File: packages/docs/getting-started/hosted-nodes.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+For more information if you're not familiar with launching EC2 instances from machine images, please check the excellent [AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html).
+
+## Google Cloud Platform
+
+To get started with a Celo blockchain node on GCP, you'll need to navigate to the Images tab of the Compute Engine page. From there you can search for public images and enter `celo-`.
+
+The cLabs GCP account is called `celo-testnet`, if you're new to Celo or GCP, ensure that the `Created by` column of the image you're launching matches `celo-testnet`.
```
Comment (barbaraliau) [inline packages/docs/getting-started/hosted-nodes.md:28]: Should we make this into a hint so it's prominent?

File: packages/docs/getting-started/running-a-full-node-in-alfajores.md [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
 This section explains how to get a full node running on the [Alfajores Network](alfajores-testnet.md), using a Docker image that was built for this purpose.
 
+If you'd prefer a one click, hosted setup for running a node on one of the major cloud providers (AWS and GCP), checkout our [hosted nodes](hosted-nodes.md) documentation.
+
 {% hint style="info" %}
 If you would like to keep up-to-date with all the news happening in the Celo community, including validation, node operation and governance, please sign up to our [Celo Signal mailing list here](https://celo.activehosted.com/f/15).
```
Comment (barbaraliau) [inline packages/docs/getting-started/running-a-full-node-in-alfajores.md]: nit: remove "here"

File: packages/terraform-modules-public/images/readme.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Celo Blockchain Images
+
+Hashicorp Packer is a tool for building machine images to ease deployment on various cloud platforms.
+
+As cLabs we provide prebuilt full and lightest Celo blockchain machine images for popular cloud platforms using this Packer script, however anyone is free to use this as inspiration for generating their own.
```
Comment (barbaraliau) [inline packages/terraform-modules-public/images/readme.md:5]: nit: ~At~ As

**Review by barbaraliau — APPROVED** [review]
Small nits. Otherwise looks good. I haven't tested any of the instructions manually but they made sense to me.

**Pull request merged**